Documentation/scaling.rst

```diff
@@ -0,0 +1,88 @@
+Scaling
+=======
+
+When working with SI units in earthquake scenarios numbers might get very large.
+Rescaling the equations might be advantageous, e.g. when working with single precision arithmetic.
+In this section, we show how to properly scale the elastic wave equation.
+
+The elastic wave equation in velocity-stress form with source terms is given by
+
+.. math::
+
+   \begin{aligned}
+     \frac{\partial\sigma_{ij}}{\partial t}
+           - \lambda \delta_{ij}\frac{\partial u_k}{\partial x_k}
+           - \mu \left(
+               \frac{\partial u_i}{\partial x_j} + \frac{\partial u_j}{\partial x_i}
+            \right) &= s_i \\
+     \rho\frac{\partial u_i}{\partial t} - \frac{\partial \sigma_{ij}}{\partial x_j} &= f_i
```

Comment:
Is s_i/f_i standard notation? Otherwise I would also mention that in the notation (e.g. with source terms s_i and f_i). Also that source terms need to be scaled.

Reply:
I added "source term" in the text.